docs/ado/guide/data/command-object-parameters.md

```diff
@@ -17,8 +17,9 @@ ms.author: jroth
 # Command Object Parameters
 The previous topic discussed [Creating and Executing a Simple Command](./creating-and-executing-a-simple-command.md). A more interesting use for the [Command](../../reference/ado-api/command-object-ado.md) object is shown in the next example, in which the SQL command has been parameterized. This modification makes it possible to reuse the command, passing in a different value for the parameter each time. Because the [Prepared Property](../../reference/ado-api/prepared-property-ado.md) property on the **Command** object is set to **true**, ADO will require the provider to compile the command specified in [CommandText](../../reference/ado-api/commandtext-property-ado.md) before executing it for the first time. It also will retain the compiled command in memory. This slows the execution of the command slightly the first time it is executed because of the overhead required to prepare it, but results in a performance gain each time the command is called thereafter. Therefore, commands should be prepared only if they will be used more than one time.  
   
-```  
+```VBA
```

Comment:
@hecon5 Please make this lowercase, and then I'll approve your change.

Reply:
Done!